showcase.json

```diff
@@ -2402,6 +2402,23 @@
     "videoContentType": "video/mp4",
     "submittableId": "33308492"
   },
+  {
+    "id": "tbfz",
+    "name": "The Battle for Zhion",
+    "website": "https://tbfz.io/",
+    "oneLiner": "Survive, conquer, and master the power of Zhion's battleground.",
+    "tags": ["Games", "Metaverse", "NFT", "Bitcoin", "Wallet"],
```

Comment:
can you please remove the Bitcoin and Wallet tags or explain why they are relevant?